dashboard/test/models/script_test.rb

```diff
@@ -2859,10 +2859,10 @@ class MigratedScriptCopyTests < ActiveSupport::TestCase
       Script.stubs(:merge_and_write_i18n)
 
       @standalone_unit = create :script, is_migrated: true, is_course: true, version_year: '2021', family_name: 'csf', name: 'standalone-2021'
-      create :course_version, content_root: @standalone_unit
+      create :course_version, :with_course_offering, content_root: @standalone_unit
 
       @unit_group = create :unit_group
-      create :course_version, content_root: @unit_group
+      create :course_version, :with_course_offering, content_root: @unit_group
```

Comment:
will https://github.com/code-dot-org/code-dot-org/pull/43955 eliminate the need for this?

Reply:
Yes -- I'll update. Thanks for catching!